### PR TITLE
Hyperledger FireFly Labs Proposal

### PR DESCRIPTION
**Pull request by nguyer**

### CODE REVIEW

**Review by lehors — CHANGES_REQUESTED** [review]
LGTM but you forgot to rename the file. You don't want to change the proposal-template.md file. Instead you want to use that file to create a new file called firefly.md
Thanks.

**Comment by nguyer** [comment]
> LGTM but you forgot to rename the file. You don't want to change the proposal-template.md file. Instead you want to use that file to create a new file called firefly.md
> Thanks.

Thanks for pointing that out! I renamed it and moved it to the `./labs` directory now.

**Review by lehors — APPROVED** [review]
LGTM. Thanks.

File: labs/firefly.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+- https://github.com/nguyer
+- https://github.com/gabriel-indik
+- https://github.com/shorsher
+- https://github.com/awrichar
+
+# Pre-existing repositories
+- https://github.com/kaleido-io/firefly
+- https://github.com/kaleido-io/firefly-cli
+- https://github.com/kaleido-io/firefly-ui
+- https://github.com/kaleido-io/ethconnect
```
Comment (tkuhrt) [inline labs/firefly.md:29]: Will we be renaming this repository to something like firefly-ethconnect?

Also, will we have resolved licensing issues with this particular repo prior to transferring into labs?

File: labs/firefly.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+FireFly is designed to be completely modular, allowing components at each layer to be swapped out for a different implementation, including the messaging system, the database, and even the blockchain itself.
+
+
+# Initial Committers
+- https://github.com/peterbroadhurst
+- https://github.com/nguyer
+- https://github.com/gabriel-indik
+- https://github.com/shorsher
+- https://github.com/awrichar
+
```
Comment (tkuhrt) [inline labs/firefly.md:24]: In the template, there is a section for Sponsor. Can you add this section and note that you do not have a sponsor?

**Review by tkuhrt — CHANGES_REQUESTED** [review]

File: labs/firefly.md (hunk 2 above)
Comment (nguyer) [reply labs/firefly.md:24]: I do not see the string "Sponsor" in this file anywhere: https://github.com/hyperledger-labs/hyperledger-labs.github.io/blob/main/proposal-template.md

Am I missing something?

File: labs/firefly.md (hunk 1 above)
Comment (nguyer) [reply labs/firefly.md:29]: Yes, we can rename this repo as a part of moving it over.

The licensing issues have been resolved in `ethconnect` now.

File: labs/firefly.md (hunk 2 above)
Comment (tkuhrt) [reply labs/firefly.md:24]: My bad. Looks like Ry removed it 4 days ago.

**Review by tkuhrt — APPROVED** [review]
LGTM

File: labs/firefly.md (hunk 2 above)
Comment (nguyer) [reply labs/firefly.md:24]: No worries! Thanks!

Comment (lehors) [reply labs/firefly.md:24]: > My bad. Looks like Ry removed it 4 days ago.

I think that's a mistake actually. We said the Sponsor was no longer required but we didn't eliminate the role.
Note: this has nothing to do with Firefly but I think the Sponsor section should be put back into the template. @ryjones ?

Comment (ryjones) [reply labs/firefly.md:24]: @lehors OK, sorry, I'm out over my skis. I'll put it back

**Pull request merged**